src/renderer/components/icon/icon.tsx

```diff
@@ -94,7 +94,7 @@ export class Icon extends React.PureComponent<IconProps> {
 
     // render as inline svg-icon
     if (svg) {
-      const svgIconText = require(`!!raw-loader!./${svg}.svg`).default;
+      const svgIconText = svg.includes("<svg") ? svg : require(`!!raw-loader!./${svg}.svg`).default;
```

Comment:
How does it work / what's fixing? 
There are some working usages of `svg` prop in codebase:

![image](https://user-images.githubusercontent.com/6377066/115860556-54a3a000-a43a-11eb-804f-84e037857d15.png)

Reply:
Those usages expect that `svg` resides within `components/icon` folder. This PR allows to pass raw svg string to `Icon` component.